lib/streamlit/caching.py

```diff
@@ -594,9 +600,12 @@ class Cache(dict):
     ...     # Fetch data from URL here, and then clean it up. Finally assign to c.
     ...     c.data = ...
 
-
     """
 
+    # Developer Note: We've changed the behavior of @st.cache to stop caching
+    # values from implicit inputs, but none of the changes in @st.cache were
+    # applied to the Cache object herein.  --@nthmost,  12/3/2019
```

Comment:
I don't really understand this comment 😕

(Also: no need for "developer note". All comments are developer notes :stuck_out_tongue: . Also no need for names and dates, since you can find them easily with git)

Reply:
KK 👍